src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const values = document.getElementsByClassName('population');
+const textNumber = [];
+
+function sortTextNumber(array) {
+  for (let i = 0; i < array.length; i++) {
+    textNumber.push(array[i].innerText);
+  }
+}
+
+function formatNumberInText(number) {
+  const options = {
+    minimumFractionDigits: 0,
+    maximumFractionDigits: 0,
+  };
+
+  return number.toLocaleString('en-US', options);
+};
+
+sortTextNumber(values);
+
```

Comment:
I don't see any reason to make a separated function. 
```suggestion
const textNumber = [];

function formatNumberInText(number) {
  const options = {
    minimumFractionDigits: 0,
    maximumFractionDigits: 0,
  };

  return number.toLocaleString('en-US', options);
};

for (let i = 0; i < values.length; i++) {
  textNumber.push(array[i].innerText);
}

```